frontend/src/pages/customers/menu/index.tsx

```diff
@@ -41,20 +44,30 @@ export default function MenuPage() {
       ) : (
         <>
           <Stack justifyContent={"space-between"} alignItems={"center"} pb={2}>
-            <Typography variant="h2" mb={2}>
-              Menu de Hoy
+            <Typography variant="h2" mb={2} textAlign={"center"}>
+              {activeMenu.name}
             </Typography>
             <Button
               variant="contained"
               sx={{ maxHeight: 50 }}
               onClick={() => router.push("/customers/order")}
             >
-              Order Now
+              Order Now!
```

Comment:
Punctuation in a button is a little weird. What do you think? 

```suggestion
              Order Now
```